virttest/test_setup.py

```diff
@@ -1496,7 +1496,7 @@ def sr_iov_setup(self):
         # Re-probe driver with proper number of VFs once more and raise
         # exception
         if re_probe:
-            if not self.remove_driver() and not self.modprobe_driver():
+            if not self.modprobe_driver():
```

Comment:
I am not faimlar ppc part and mlx5_core driver well.
"remove_driver"  here is used to disable all of the vfs (vf_no=0) before modprobe pf driver.
I think it is ok here. no need to be removed.

Reply:
Before commit  af59aa5797c8c9fcb569914dfeffd17c872242bd,  the function "remove_driver" only disable vf (set vf num to 0) for ppc and mlx5_core, i guess this is need for specific card. In that commit, x86 was added to disable vf. this is not must for x86, but it work too. So, consider ppc+mlx5_core, this function can not be removed here. And, before modprobe_driver,  remove_driver again even if it is not exist is ok.  it will return True when remove a no-exist driver.

Reply:
Yes, it cannot be removed. But there is something wrong with the logic. I will update.  Thanks!

Reply:
@yanan-fu right!